### ISSUE TITLE
Body background not reset on drawer close

### ISSUE THREAD

**Issue by andrewdoro** [emilkowalski/vaul#188]
https://github.com/emilkowalski/vaul/blob/25e4999b0afd92b4b47d35a6a56cb16074610ce8/src/index.tsx#L394C1-L395C1
Using 0.8
## Steps to reproduce
1. Go to https://vaul.emilkowal.ski/
2. Open drawer and then close it
4. Look at the body tag styles, background is still set to black
<img width="634" alt="Screenshot 2023-12-26 at 16 00 08" src="https://github.com/emilkowalski/vaul/assets/17547035/75133ad0-35b3-4378-8dd4-640198bc64e1">

**Comment by yigityalim** [comment]
Yes! This issue needs to be resolved quickly!

**Comment by dseiner** [comment]
I'm facing the same issue, but it only occurs when I add the "vaul-drawer-wrapper" div.

```
<div data-vaul-drawer-wrapper="">
{children}
</div>
```

The problem arises when I use the scale background feature; the background color resets upon drawer close. However, I want to use this feature.

As a workaround, I've used:
```
<div data-vaul-drawer-wrapper="" className="bg-background min-h-[100vh]">
{children}
</div>
```

This way, the background color remains the default instead of being replaced by black, as injected by the drawer. But still not optimal.

**Comment by Mittsuyama** [comment]
This works for me.

**Comment by Mittsuyama** [comment]
This works for me:

```tsx
<Drawer onClose={() => document.body.style.background = ''}>
  {/* ... */}
</Drawer>
```

**Comment by choeqq** [comment]
solved it here: 
https://github.com/emilkowalski/vaul/pull/232

**Comment by emilkowalski** [comment]
Fixed in #232.
